pyproject.toml

```diff
@@ -42,5 +42,5 @@ line_length = 88
 known_first_party = "statmake"
 
 [build-system]
-requires = ["poetry>=1.0.0"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core"]
```

Comment:
```suggestion
requires = ["poetry-core>=1.0.0"]
```

Reply:
That should not be necessary according to https://pypi.org/project/poetry-core/

Reply:
Mh... So it seems. I took this from what poetry puts into a new project, but looking at the release hsitory of -core, this probably isn't necessary. Ok, will take as is. Thanks!